chain-core/src/init/address.rs

```diff
@@ -51,7 +51,7 @@ where
 #[derive(Debug)]
 pub enum ErrorAddress {
     /// An invalid length
-    InvalidLength(usize),
+    InvalidLength(u64),
```

Comment:
I think this one isn't necessary (not implementing parity scale codec)

Reply:
done